metricshub-web/react/src/components/test-button.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Button } from "@mui/material";
+
+export default function TestButton() {
+	return (
+		<Button
+			variant="contained"
+			disableRipple
+			aria-hidden
+			sx={{
```

Comment:
Button sizes shouldn't be specified for each button.

But this TestButton should not be here, right?